src/foundry/common/constants.d.mts

```diff
@@ -164,16 +164,14 @@ export declare const CHAT_MESSAGE_STYLES: Readonly<{
   EMOTE: 3 & CHAT_MESSAGE_STYLES;
 
   /**
-   * @deprecated since v12
-   *
-   * @remarks Define rolls directly in ChatMessage#rolls instead.
+   * @deprecated "`CONST.CHAT_MESSAGE_STYLES.ROLL` is deprecated in favor of defining rolls directly in {@linkcode ChatMessage.rolls | ChatMessage#rolls}"
+   * (since v12, until v14)
    */
   ROLL: 0 & CHAT_MESSAGE_STYLES;
 
   /**
-   * @deprecated since v12
-   *
-   * @remarks Define whispers directly in ChatMessage#whisper instead.
+   * @deprecated "`CONST.CHAT_MESSAGE_STYLES.Whisper` is deprecated in favor of defining whisper recipients directly in {@linkcode ChatMessage.whisper | ChatMessage#whisper}"
+   * (since v12, until v14)
```

Comment:
Can we put this on one line? As I pointed out a whiel ago ESLint's no-deprecated only shows the first line and I want `(since v12, until v14)` in the message ideally.